docs/docsource/installation/upgrade/upgrade_zenko.rst

```diff
@@ -63,18 +63,20 @@ To upgrade Zenko:
 #. From the kubernetes/ directory of the new Zenko source, enter this
    Helm command, inserting your Zenko server's name:
 
-   :: 
-
+   ::
+      
       $ helm upgrade {{zenko-server-name}} ./zenko
 
    If you are using custom values, reuse options.yaml on upgrades.
+   
    ::
 
-      $ helm upgrade {{zenko-server-name}} ./zenko -f options.yaml
+      $ helm upgrade {{zenko-server-name}} . -f options.yaml
```

Comment:
I think this line is correct and should not be changed.

Reply:
I agree. Will roll it back. Thanks.

Reply:
Double-checking, I see the folllowing text at: installation/configure/configuring_zenko.html

"The options.yaml file is not present by default, but you added a simple one at Zenko/kubernetes/options.yaml when deploying Zenko."

The problem here is that in 1.1, we recommended Zenko/kubernetes/zenko/options.yaml. 
In 1.2 we're recommending a new structure: Zenko/kubernetes/options.yaml. 

Xavier caught a lot of these. Please review this to make sure I'm not crazy.

Reply:
I agree with this, though the line as it is now looks correct, right? So starting from the `kubernetes` directory, we are upgrading the `./zenko` directory, using `./options.yaml`, which looks correct to me.

Reply:
Also you can compare with the `helm upgrade` line above at line 68.

Reply:
Okay, that's persuasive. I am making the requested change. Sorry about the confusion.